view/frontend/web/js/components/BaseButton.vue

```diff
@@ -1,22 +1,53 @@
 <template>
-  <button :type="buttonType" :title="buttonTitle">
-    {{ text }}
+  <button
+    class="button"
+    :type="buttonType"
+    :title="buttonTitle"
+    :disabled="isBlocked"
+  >
+    <div class="button__handler">
+      <span class="button__text">
+        {{ text }}
+      </span>
+
+      <template v-if="withLoader">
+        <BaseLoader/>
+      </template>
```

Comment:
Can you move `v-if="withLoader"` to BaseLoader

Reply:
I can say more. I removed this v-if b/c found it's unecessary

Reply:
nope

Reply:
b/c it's variant for button. Not always you want to show button with loader. I.e when have 2 buttons and one is "Place order" and the second is "Back"

Reply:
But it does not metter you can move condition to BaseLoader, like below
```
<BaseLoader  v-if="withLoader"/>
```
in this case we dont need temlate tag